src/lib/appconfig.rs

```diff
@@ -91,8 +91,10 @@ pub fn run_server(
     listener: TcpListener,
     db_pool: PgPool,
 ) -> Result<Server, std::io::Error> {
-    let db_pool = Data::new(db_pool);
     let server = HttpServer::new(move || {
+        let db_pool_d = Data::new(db_pool.clone());
+        let settings_d = Data::new(settings.clone());
+        let statsd_d = Data::new(StatsD::new(&settings));
```

Comment:
The fact that you are instantiating a new statsd client here rather than pulling in the one we init in the `CJ` struct makes me think that there is something going on here that requires a separate client instance per thread. Is this the case? If so, would it be worth calling out with a comment?

Reply:
Potentially relevant: https://docs.rs/cadence/latest/cadence/struct.StatsdClient.html#wrapping-with-an-arc